internal/dialect/dialectquery/starrocks.go

```diff
@@ -0,0 +1,45 @@
+package dialectquery
+
+import "fmt"
+
+type Starrocks struct{}
+
+var _ Querier = (*Starrocks)(nil)
+
+func (m *Starrocks) CreateTable(tableName string) string {
+	q := `CREATE TABLE IF NOT EXISTS %s (
+		version_id bigint NOT NULL,
+		id bigint NOT NULL AUTO_INCREMENT,
+		is_applied boolean NOT NULL,
+		tstamp datetime NULL default CURRENT_TIMESTAMP
+	)
+	PRIMARY KEY (version_id,id)
+	DISTRIBUTED BY HASH (id)
+	ORDER BY (version_id)`
```

Comment:
Sorry, 2 questions:

1. Should it ORDER BY (id)
2. Should PRIMARY KEY be (id) instead of (version_id,id)?

Reply:
Yes, fixed